python/py-pyobjc-fsevents/Portfile

```diff
@@ -1,59 +1,24 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python  1.0
+PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                py-pyobjc-fsevents
-python.rootname     pyobjc-framework-FSEvents
 version             5.3
 revision            0
-categories-append   devel
-platforms           darwin
-license             MIT
-maintainers         nomaintainer
-
-description         FSEvents wrapper for PyObjC
-long_description    This port provides the FSEvents wrapper for use with PyObjC
-homepage            https://pyobjc.readthedocs.io/
-
-master_sites        pypi:p/${python.rootname}/
-distname            ${python.rootname}-${version}
-
-checksums           rmd160  c54b7ad3dfb4e977504e9028a4ca81a07ae9b869 \
-                    sha256  b5570c4184c75690b3f8e5ed5034021369ae664f33539dc85e41c3a6c7e6d88c \
-                    size    25769
-
-python.versions     27 35 36 37 38
-
-if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools
-    depends_lib-append  port:py${python.version}-pyobjc
+epoch               1
 
-    patchfiles      pyobjc_setup.py.patch
+categories          python devel
+license             MIT
 
-    if {${configure.sdkroot} eq ""} {
-        set sdkroot "/"
-    } else {
-        set sdkroot ${configure.sdkroot}
-    }
+replaced_by         py-pyobjc
 
-    build.env-append    SDKROOT=${sdkroot}
-    destroot.env-append SDKROOT=${sdkroot}
+description         obsolete wrapper for PyObjC
 
-    platform darwin 8 {
-        depends_build
-        depends_lib
-        pre-fetch {
-            ui_error "${subport} @${version} requires Mac OS X 10.5 or later"
-            return -code error "incompatible Mac OS X version"
-        }
+foreach v {27 35 36 37 38} {
```

Comment:
please remove the py35 subport; you cannot set it to be replaced_by py35-pyobjc as that port doesn't define the py35 subport